free-2.md

```diff
@@ -58,12 +105,28 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since I'd think most of the work goes into the selector,
+I don't think the downloading feature would require very much implementation;
+so I'd say ~20% of the time would be spent engaging in the language aspects.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The idea doesn't seem too big or ambitious, so I think it's reasonable to expect completion
+of the project along with a moderate amount of attention given to error and warning messages.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+**Benefits**
+* Seems reasonable to expect completion by the semester's end, which allows time to develop a well-rounded yet brief language
+
+**Drawbacks**
+* Small language likely means that not a lot of tradeoffs will be considered.
+
+
+[jQuery]: https://en.wikipedia.org/wiki/JQuery
+[Downloadify]: https://github.com/dcneiner/Downloadify
```

Comment:
I think this idea is interesting, though it would definitely need quite a bit more detail before it became a project. Another potentially related idea is [Beautiful Soup](http://www.crummy.com/software/BeautifulSoup/), which is a Python library for parsing HTML files.